compiler/compile/OMRCompilation.cpp

```diff
@@ -740,38 +749,78 @@ bool OMR::Compilation::isPotentialOSRPointWithSupport(TR::TreeTop *tt)
    return potentialOSRPoint;
    }
 
+/*
+ * There are two OSR modes: voluntary and involuntary OSR.
+ * In voluntary OSR mode, the JIT is aware of where OSR transitions will occur
+ * and can manage them. In involuntary OSR mode, the OSR transition point is not known
+ * and can occur at any point.
```

Comment:
This should read something more like: In involuntary OSR mode, the JITed code does not control when an OSR transition occurs - a transfer can be initiated externally at any potential OSR point.